samples/workload/tensorflow/CMakeLists.txt

```diff
@@ -0,0 +1,76 @@
+# Copyright (C) 2019 Intel Corporation. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+cmake_minimum_required (VERSION 3.14)
+
+project(tflite_wasm)
+
+include(ExternalProject)
+
+# Only Download
+ExternalProject_Add(tensorflow-download
+    PREFIX tensorflow
+    GIT_REPOSITORY https://github.com/tensorflow/tensorflow.git
+    GIT_TAG        v2.13.0-rc2
+    GIT_PROGRESS   ON
+    SOURCE_DIR     ${CMAKE_CURRENT_SOURCE_DIR}/tensorflow
+    UPDATE_COMMAND git restore .
+    PATCH_COMMAND  git apply ${CMAKE_CURRENT_SOURCE_DIR}/tflite_bazel_building.patch
+                     && ${CMAKE_COMMAND} -E copy_if_different ${CMAKE_CURRENT_SOURCE_DIR}/bazelrc.user ${CMAKE_CURRENT_SOURCE_DIR}/tensorflow/.bazelrc.user
```

Comment:
The file `bazelrc.user` wasn't uploaded?

Reply:
Just found it. Have uploaded. 👍